FtcRobotController/src/main/java/org/firstinspires/ftc/robotcontroller/external/samples/HopperAPI.java

```diff
@@ -0,0 +1,57 @@
+package org.firstinspires.ftc.robotcontroller.external.samples;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.ColorSensor;
+
+/**
+ * Created by casciarobotics on 10/6/16.
+ */
+public class HopperAPI {
```

Comment:
This looks good. If we have to change from motors to servos, we won't have to change any of the other code (outside of this file).